ntoskrnl/mm/i386/page.c

```diff
@@ -560,7 +598,7 @@ MmCreatePageFileMapping(PEPROCESS Process,
     /* And we don't support creating for other process */
     ASSERT(Process == PsGetCurrentProcess());
 
-    if (SwapEntry & (1 << 31))
+    if (SwapEntry & (1 << (sizeof(SWAPENTRY) - 1)))
```

Comment:
Same problem might also be present there:
https://git.reactos.org/?p=reactos.git;a=blob;f=ntoskrnl/mm/i386/pagepae.c;hb=59e74584ac5c48e9edb2bf494c2224bd5ca6fa90#l1072